dashboard/app/models/contact_rollups_processed.rb

```diff
@@ -26,18 +26,27 @@ class ContactRollupsProcessed < ApplicationRecord
   DATA_UPDATED_AT_KEY = 'u'.freeze
 
   # Aggregates data from contact_rollups_raw table and saves the results, one row per email.
-  # @param [Integer] batch_size number of records to save per INSERT statement.
+  # @param batch_size [Integer] number of records to save per INSERT statement.
+  # @return [Hash] number of valid and invalid contacts (emails) in the raw table
   def self.import_from_raw_table(batch_size = DEFAULT_BATCH_SIZE)
+    valid_contacts = 0
+    invalid_contacts = 0
+
     # Process the aggregated data row by row and save the results to DB in batches.
     batch = []
     ContactRollupsV2.retrieve_query_results(get_data_aggregation_query).each do |contact|
-      contact.deep_stringify_keys!
-      contact_data = parse_contact_data(contact['all_data_and_metadata'])
+      begin
+        contact.deep_stringify_keys!
+        contact_data = parse_contact_data(contact['all_data_and_metadata'])
+        valid_contacts += 1
+      rescue StandardError
```

Comment:
What situation(s) would produce invalid contacts?

Reply:
This is the result from the adhoc:
```ruby
ContactRollupsProcessed.import_from_raw_table
=> {:valid_contacts=>4123589, :invalid_contacts=>118500}
```

1 specific error I saw was `ArgumentError (mon out of range)` caused by failing to parse Time value.

Reply:
Just noting from offline conversation that Ha notes that it might be useful to store intermediate results somewhere so we can debug what is wrong with these 118K contacts.